### PR TITLE
Added ixgbe to bromolow models

### PR DESCRIPTION
**Pull request by fbelavenuto**

### CODE REVIEW

**Comment by nemesis122** [comment]
Hi Fabio
I have tested 3615 with 113a without updateing this ixgbe modules that you have changed here see below
[Added ixgbe to bromolow models](https://github.com/fbelavenuto/arpl-modules/pull/174/commits/61649e3591eedf04bb9f2e542eb6e8db0f681ce3)
then all is working

BUT when i updated the modules and create 3615 whith your new ixgbe driver added for bromolow models 3615  then 
 the install with 113a Model 3615 is failed 
Added ixgbe to bromolow models #174 <----

In short this change is wrong because the ixgbe driver is broken for 3615xs when this change is added 
Added ixgbe to bromolow models #174

without this change 3615xs ixgbe Intel 520 driver is working for the bromlow Modell 3615 with the loader 113a
